tools/src/Workspace.ts

```diff
@@ -25,12 +32,39 @@ export type WorkspacesInfo = Record<string, WorkspaceProjectInfo>;
  * Returns an object containing info for all projects in the workspace.
  */
 export async function getInfoAsync(): Promise<WorkspacesInfo> {
+  // This ony lists workspace dependencies from `dependencies` and `devDependencies`.
```

Comment:
```suggestion
  // This only lists workspace dependencies from `dependencies` and `devDependencies`.
```

nit: did you mean "only"?

Reply:
Yes, thanks for catching that!